config/config-example.js

```diff
@@ -379,3 +379,10 @@ exports.grouplist = [
 		ip: 's',
 	},
 ];
+
+// punishgroups - the symbols used to mark the different punishment groups, locked and muted
```

Comment:
I think declaring it in config.js and having an extra "catch" in rooms.js (for building the list) would be best since that's where the custom rank list is built